InventoryManagement/InventoryManagement/Models/Product.cs

```diff
@@ -5,14 +5,22 @@ namespace InventoryManagement.Models
     public class Product
     {
         public int Id { get; set; }
+
+        // CODE REVIEW CHANGE: Added [Required] to enforce that
+        // every product must have a UPC code. This improves data integrity.
```

Comment:
This is a no-no.  Don't put conversation/justification in the source code.  That's clutter.  Clutter is technical debt.
Conversation and justification is fine, but it belongs in the Files Changed section in the Pull Request, where I am putting this now.  That allows you to see and participate in the conversation, with specific source code in context, without cluttering the code.